### ISSUE TITLE
Replacement of icons in "treatments" section

### ISSUE THREAD

**Issue by virsto** [obaez/dentistsmile#3]
I have not found a method to replace the icons used in the treatments section. It seems that the source of the problem  is the processing of the svg-defs.svg file in _includes. That is, this file defines the icons used regardless of how the icons are defined in the svg folder. The problem could probably be resolved with information how to generate the svg-defs.svg file for icons defined by the user.

**Comment by obaez** [comment]
The svg-defs are being included through the Gruntfile. If you add any svg icons to the svg folder and run grunt, they'll be added to the svg-defs and can be used on the theme.

**Comment by virsto** [comment]
On 2016-Feb-22 14:53, Orlando wrote:
>
> The svg-defs are being included through the Gruntfile. If you add any svg 
> icons to the svg folder and run grunt, they'll be added to the svg-defs and 
> can be used on the theme.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/obaez/dentistsmile/issues/3#issuecomment-187185142>.
>
Thank you Orlando for your prompt response.

I learned a lot from studying your email.

  1) I needed a local (to dentistsmile folder) installation of npm and grunt
  2) I needed to execute the following:

...>grunt svgstore   [This was something new for me!]

  Then "eureka" the _includes/svg-defs.svg file was created (from the contents 
of  the folder svg) which is exactly what I hoped for.

This worked fine on my Windows Vista system. I will see what I can do on my 
Windows 10 system later tonight.

Thanks again Orlando and have a good day.

--V :-)
